charts/kyuubi/values.yaml

```diff
@@ -101,17 +101,19 @@ server:
 kyuubiConfDir: /opt/kyuubi/conf
 kyuubiConf:
   # The value (templated string) is used for kyuubi-env.sh file
-  # See https://kyuubi.apache.org/docs/latest/deployment/settings.html#environments for more details
+  # See https://kyuubi.readthedocs.io/en/master/deployment/settings.html#environments for more details
   kyuubiEnv: ~
 
   # The value (templated string) is used for kyuubi-defaults.conf file
-  # See https://kyuubi.apache.org/docs/latest/deployment/settings.html#kyuubi-configurations for more details
-  kyuubiDefaults: ~
+  # See https://kyuubi.readthedocs.io/en/master/deployment/settings.html#kyuubi-configurations for more details
+  kyuubiDefaults: |
```

Comment:
it should be good to keep `~`, but let's add more comments, including "examples"

Reply:
Thanks! I add Example usecases in kyuubiConf.